### DISCUSSION TITLE
I made a bash script to generate TTS playback of long text seamlessly

### DISCUSSION THREAD

**Discussion by JJenkx** [coqui-ai/TTS#1622] [Show and tell]
I made this script to allow playback of long bodies of text to listen as it is being encoded. Sharing in case anyone has use for it

https://github.com/JJenkx/Personal/blob/main/tts.sh

or raw

https://raw.githubusercontent.com/JJenkx/Personal/main/tts.sh

**Comment by Milor123** [comment]
Thank you very much works very nice, I have a question, i use Spanish lenguague buts this script delete my "acute accent", should do convert it in normal words? or the acute accent are necesary for a best result of coqui tts for spanish lenguage?

**Comment by Milor123** [comment]
I have a problem bro.

```
 > Saving output to /home/noe/Documents/tts/000000.temp.wav
./tts.sh: line 179: parallel: command not found
nohup: redirecting stderr to stdout

```
